Documenting new feature :step_name :bookmark: (#811)
fastlane/docs: docs/advanced/actions.md

# Actions

# Naming actions

> Starting from `fastlane@v2.120` you have now the ability to specify Step Name for all actions.

Reviewer: ```suggestion
You can specify a "step name" for all actions (starting with `fastlane 2.120.0`).
```

> This was introduced to disambiguate the use of some action several times, in different context.

Reviewer: ```suggestion
```

To do so, add the `:step_name` property to your arguments.

For instance with the [fastlane-plugin-yarn](https://github.com/joshrlesch/fastlane-plugin-yarn):
```ruby
yarn(commmand: 'build', step_name: 'building_project')
yarn(commmand: 'test', step_name: 'testing_project')
yarn(commmand: 'publish-coverage', step_name: 'publishing_project')
```

> These name will appear as the step description instead of generic `yarn`.

Reviewer: Add the generated output from the command line here so it is easier to visualize the result.
